ocs_ci/framework/conf/default_config.yaml

```diff
@@ -97,7 +97,6 @@ ENV_DATA:
   worker_replicas: 3
   skip_ocp_deployment: false
   skip_ocs_deployment: false
-  logging_version: '4.3'
```

Comment:
You are also missing those files:
conf/ocs_version/ocs-4.2.yaml:  logging_version: '4.2'
conf/ocs_version/ocs-4.3.yaml:  logging_version: '4.3'

Reply:
Removed